README.md

```diff
@@ -72,6 +72,43 @@ npm run precommit
 - Sync complete `build` directory on server. 
 - Before code push to repository, make sure you lint your code using `npm run precommit` command.
 
+## Contributing
```

Comment:
We slightly updated our contributor's workflow content, To align with that please update flow here. You can refer https://github.com/rtCamp/edit-flow-slack-integration